mysite/blog/tests/test_tags.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from blog.templatetags import blog_tags
+from blog.models import Post, Comment
+from blog.tests.test_ModelMixinTestCase import ModelMixinTestCase
+
+
+class TestTemplateTags(ModelMixinTestCase, TestCase):
+    def test_count_tag(self):

```

Comment:
test_action_provides_result should be the test case name. If not able to name like that, the test case is not correct